### DISCUSSION TITLE
La liebre y la tortuga

### DISCUSSION THREAD

**Discussion by roverem** [gamerevar/proyecto-comunitario-1-gd#49] [Archivados]
Premisa: "la idea es que hay dos personajes que tienen que ir a un lugar por una buena razón, y al final se tienen que enfrentar y el que gana es el que consigue/cumple su buena razón. Los dos empiezan de lugares diferentes y tienen caminos diferentes, pero terminan en el mismo lugar"

La idea me remite inmediatamente a la fabula de la liebre y la tortuga

![image](https://user-images.githubusercontent.com/7843438/101815818-e56a8680-3afe-11eb-857a-13312af92b04.png)

Se me ocurre que se le puede buscar muchas versiones a la temática. Imaginando rapidamente algunas opciones, una versión cyberpunk, una versión clásica (cartoons), una versión con personajes antropomórficos que viven en sociedad (donde los animales tienen trabajos, por ejemplo).

En cuanto a gameplay se me ocurren algunas ideas:

- Un runner de upgrades (burrito bison, por ej) Donde el jugador hace runs para comprar upgrades para eventualmente ganarle a la liebre.
- Una aventura de diálogo, donde la tortuga puede interactuar con personajes y el entorno para conseguir ganarle a la liebre.

Mas adelante si hay interés podría agregar o ampliar las ideas

**Comment by jh62** [comment]
No soy fana de los runner, pero creo que es fácilmente aplicable a la premisa. Cómo se podría expandir la idea para que no sea muy chicle?

Un juego de diálogos pura y exclusivamente no me convence. Si es un point and click sería más interesante, aunque con otro setting diferente al de la tortuga y la liebre.

Veamos que opina el resto.

**Reply by roverem** [reply]
Quizás no necesariamente tenga que ser un runner pero dentro de la premisa me parece piola esto de que el jugador participe de un loop corto, que tenga que realizar muchas veces que solo va poder ganar una vez que lo haya realizado una determinada cantidad de veces mínimas (por conseguir upgrades afuera del loop o por entender algo que sucede dentro del loop que la primera vez que lo vió no supo como resolver).

Ejemplo concreto:

Jugador debe realizar una torta.
El primer run sale una torta básica con harina, huevos y leche (que son los ingredientes que tiene a disposición).
Cada torta le da una cantidad de puntos a gastar en un menú que se le presenta entre loops.
Luego de 5 runs puede comprar con esos puntos chocolate, entonces completar la torta le da mas puntos.

Puse un ejemplo no runner a proposito porque es una idea que se puede abstraer facil y aplicar en muchas cosas
